src/ChunkSender.cpp

```diff
@@ -89,7 +89,7 @@ void cChunkSender::Stop(void)
 
 
 
-void cChunkSender::QueueSendChunkTo(int a_ChunkX, int a_ChunkZ, eChunkPriority a_Priority, cClientHandle * a_Client)
+void cChunkSender::QueueSendChunkTo(int a_ChunkX, int a_ChunkZ, Priority a_Priority, cClientHandle * a_Client)
```

Comment:
This function looks like it should behave wrongly with the new priority system. As is, it looks like queueing a chunk with lower priority than it already has will be taken forwards. At the very least, this needs a comment to explain what it's doing.